versions/v1/units.json

```
@@ -1179,6 +1179,65 @@
     "source": "qudt.org",
     "sourceReference": "https://qudt.org/vocab/unit/MilliFARAD"
   },
+  {
+    "externalId": "density:gm-per-centim3",
+    "name": "GM-PER-CentiM3",
+    "quantity": "Density",
+    "longName": "Gram Per Cubic Centimeter",
+    "aliasNames": [
+      "Gram per Cubic Centimeter",
+      "gram per CM3",
+      "G PER CM3",
+      "g per CM3",
+      "g / Cubic Centimeter",
+      "Gram / Cubic Centimeter",
+      "Gram / cubic centimeter",
+      "g / cm³",
+      "gram / cubic centimeter",
+      "Gram per cubic centimeter",
+      "Gram per CM3",
+      "gram per cubic centimeter",
+      "GM per cm³",
+      "gram per cubic centimeter",
+      "GM / Cubic centimeter",
+      "g / CM3",
+      "g per cubic centimeter",
+      "GM per cubic centimeter",
+      "gram / Cubic centimeter",
+      "GM / cubic centimeter",
+      "GM / CM3",
+      "GM per CM3",
+      "g per Cubic Centimeter",
+      "GM per cubic centimeter",
+      "GM / cm³",
+      "Gram Per Cubic Centimeter",
+      "GM / cubic centimeter",
+      "g / cubic centimeter",
+      "Gram per cm³",
+      "gram per cm³",
+      "gram / cm³",
+      "Gram / cubic centimeter",
+      "g per cm³",
+      "g / cubic centimeter",
+      "gram / CM3",
+      "g per cubic centimeter",
+      "gram / cubic centimeter",
+      "gram per Cubic Centimeter",
+      "GM PER CM3",
+      "GM per Cubic Centimeter",
+      "Gram Per Cubic Centimeter",
+      "Gram / CM3",
+      "Gram per cubic centimeter",
+      "Gram / cm³"
+    ],
```

You have duplicated aliases in the list. Please remove them as suggested below:

```suggestion
    "aliasNames": [
      "Gram per Cubic Centimeter",
      "gram per CM3",
      "G PER CM3",
      "g per CM3",
      "g / Cubic Centimeter",
      "Gram / Cubic Centimeter",
      "Gram / cubic centimeter",
      "g / cm³",
      "gram / cubic centimeter",
      "Gram per cubic centimeter",
      "Gram per CM3",
      "gram per cubic centimeter",
      "GM per cm³",
      "GM / Cubic centimeter",
      "g / CM3",
      "g per cubic centimeter",
      "GM per cubic centimeter",
      "gram / Cubic centimeter",
      "GM / cubic centimeter",
      "GM / CM3",
      "GM per CM3",
      "g per Cubic Centimeter",
      "GM / cm³",
      "Gram Per Cubic Centimeter",
      "g / cubic centimeter",
      "Gram per cm³",
      "gram per cm³",
      "gram / cm³",
      "g per cm³",
      "gram / CM3",
      "gram per Cubic Centimeter",
      "GM PER CM3",
      "GM per Cubic Centimeter",
      "Gram / CM3",
      "Gram / cm³"
    ],
```